code/datums/commodity.dm

```diff
@@ -1893,6 +1893,30 @@
 	upperfluc = 100
 	lowerfluc = -100
 
+/datum/commodity/medical/firstaidC
+	comname = "First Aid Kit - Critical"
+	comtype = /obj/item/storage/firstaid/crit
+	price = 2000
+	baseprice = 2000
```

Comment:
I'd raise the price on this a bit and remove it from the QM crate to reinforce status as a high-end luxury medical item - maybe add as a military-locked debris crate loot option?